wordpress-mobile/WordPress-Android
# Revert "Issue 12404/remove status and visibility option"

Fixes #12404 

## Solution
This PR reverts the changes that were made in wordpress-mobile/WordPress-Android#12409 to disable the Status & Visibility option in the bottom sheet. This revert is taking place because we will be merging these changes into the 15.3 instead of the 15.2 release that this PR currently targets. 

## Testing

Ensure the Status & Visibility option as seen in the screenshot below exists within the current release. 

<kbd><img src="https://user-images.githubusercontent.com/1509205/87189446-44a93e00-c2b6-11ea-9a0b-3e407041349c.png" width="320"></kbd>

---------------

1. Create a new post. 
2. Click Publish. 
3. Ensure that the Status & Visibility option is visible. 

------------
1. Create a new page. 
2. Click Publish. 
3. Ensure that the Status & Visibility option is visible. 

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
